src/ansible_creator/resources/new_collection/tox-ansible.ini.j2

```diff
@@ -3,6 +3,7 @@
 skip =
     py3.7
     py3.8
+    py3.9
```

Comment:
The scaffolded collection would still support Py 3.9 because stable-2.14 isn't EoL yet.